reports/trr0000/win/README.md

```diff
@@ -0,0 +1,249 @@
+
+- KerbQueryTicketCacheMessage - queries the local ticket cache
+- KerbRetrieveEncodedTicketMessage - retrieves a ticket from the local
+  cache
+- KerbSubmitTicketMessage - submits a ticket to be included in the local cache
+
+> [!NOTE]
+> This list of messages is not necessarily inclusive. It's probably possible to
+> accomplish the same task with some of the other supported messages, like
+> KerbQueryTicketCacheExMessage or KerbAddExtraCredentialsMessage.
+
+Most of the Kerberos SSP/AP routines require a calling process to hold the
+`SeTcbPrivilege` ('Trusted Computer Base'), which indicates the process is a
+trusted part of the operating system. A process running as `SYSTEM` is allowed
+to enable this privilege.
+
+### The Kerberos Ticket Cache
+
+Because Kerberos is stateless, it requires a client to hold valid tickets in a
+local ticket cache so they can be resubmitted when a client needs to request a
+new service ticket or request access to a service. On Windows, this cache is
+managed by the LSA. Each logon session has its own ticket cache. Windows comes
+with a [klist] utility that allows users to list logon sessions and their
+associated tickets. Users can only view their own sessions and tickets, but
+accounts holding `Administrator` privileges are permitted to list other user's
+sessions and tickets. Accounts holding `SeTcbPrivilege` can extract tickets from
+any logon session.
+
+## Procedures
+
+| ID              | Title                          | Tactic            |
+|-----------------|--------------------------------|-------------------|
+| TRR0000.WIN.A   | Submit ticket to local cache | Defense Evasion, Lateral Movement |
+
+### Procedure A: Submit ticket to local cache
+
+Once a ticket has been stolen, it can be injected into any logon session using
+the Kerberos SSP/AP `KerbSubmitTicketMessage` message. This submits the ticket
+to be included in the current logon session's ticket cache, where it will be
+used whenever access is requested. One important detail to note here: it is
+pointless for an attacker to inject a ticket for a user into a logon session for
+that same user, because it's a simple matter to request a new valid ticket for
+the logged on user. **Thus, maliciously injected tickets will always have a
+mismatch between the client the ticket was issued for and the logon session user
+account.**
+
+#### Detection Data Model
+
+![DDM - Procedure Name 1](ddms/ddm_trr0000_win_a.png)
+
+#### A Note on Ticket Acquisition
+
+As noted in the scope statement, there are many different ways an attacker could
+potentially acquire valid Kerberos tickets that could be used in a
+pass-the-ticket attack. The DDM includes the ones most commonly used with this
+procedure, like those used by the popular tools [Rubeus] and [Mimikatz]. The DDM
+also acknowledges that there are other ways, but this TRR will not attempt an
+exhaustive exploration of them. All methods for acquiring tickets are in gray on
```

Comment:
```suggestion
also acknowledges that there are other ways, but this TRR will not attempt an
exhaustive exploration of them. All methods in gray are for acquiring tickets on
```
"not exhaustive... All methods...". I knew what you meant, but it was a little unclear grammatically.